release-tool

```diff
@@ -575,6 +589,10 @@ build() {
     for p in $BUILD_PLUGINS; do
         CMAKE_OPTIONS="${CMAKE_OPTIONS} -DWITH_XC_$(echo $p | tr '[:lower:]' '[:upper:]')=On"
     done
+
+    if $BUILD_NIGHTLY; then
+        CMAKE_OPTIONS="${CMAKE_OPTIOS} -DKEEPASSXC_NIGHTLY_BUILD=On -DKEEPASSXC_NIGHTLY_VERSION=${RELEASE_NAME}"
```

Comment:
@droidmonkey `CMAKE_OPTIOS` -> `CMAKE_OPTIONS`?

Reply:
Thats the one